Presentation.Web/Controllers/API/V1/ItContractController.cs

```diff
@@ -388,5 +409,15 @@ public override HttpResponseMessage Post(int organizationId, ItContractDTO dto)
                 .Create(organizationId, dto.Name)
                 .Match(NewObjectCreated, FromOperationError);
         }
+
+        private static IEnumerable<OptionWithDescriptionAndExpirationDTO> ToDTOs<T>(IEnumerable<(OptionDescriptor<T> option, bool available)> options, int organizationId) where T : OptionEntity<ItContract>
+        {
+            return options.Select(ToDTO);
+        }
+
+        private static OptionWithDescriptionAndExpirationDTO ToDTO<T>((OptionDescriptor<T> option, bool available) optionObject) where T : OptionEntity<ItContract>
+        {
+            return new OptionWithDescriptionAndExpirationDTO(optionObject.option.Option.Id, optionObject.option.Option.Name, optionObject.available, optionObject.option.Description);
```

Comment:
Notice.. you are putting available into an "expired" argument. It must be `!available`